packages/next/test/fixtures/00-middleware/middleware.js

```diff
@@ -47,6 +47,11 @@ export function middleware(request) {
     return;
   }
 
+  if (url.pathname === '/somewhere') {
```

Comment:
How about a test to request the `/somewhere` path?

Reply:
Added in https://github.com/vercel/vercel/pull/7912/commits/658bbb8187c5db20ed39549272661ab8df6bfa56